app/components/land-use-chart.js

```diff
@@ -0,0 +1,152 @@
+import Ember from 'ember'; // eslint-disable-line
+import ResizeAware from 'ember-resize/mixins/resize-aware'; // eslint-disable-line
+
+import carto from '../utils/carto';
+import landUseColors from '../utils/landUseColors';
+
+const LandUseChart = Ember.Component.extend(ResizeAware, {
+  resizeWidthSensitive: true,
+  resizeHeightSensitive: true,
+
+  debouncedDidResize() {
+    this.createChart();
+  },
+
+  borocd: '',
+  sql: Ember.computed('borocd', function sql() {
+    const borocd = this.get('borocd');
+    const SQL = `
+      WITH lots AS (
+        SELECT a.the_geom, c.description as landuse_desc, c.code as landuse
+        FROM support_mappluto a
+        INNER JOIN support_landuse_lookup c
+              ON a.landuse::integer = c.code
+        INNER JOIN support_admin_cdboundaries b
+        ON ST_Contains(b.the_geom, a.the_geom)
+
+        AND b.borocd = '${borocd}'
+      ),
+      totalsm AS (
+        SELECT sum(ST_Area(the_geom::geography)) as total
+        FROM lots
+      )
+
+      SELECT count(landuse), sum(ST_Area(the_geom::geography)) * 10.7639 as sqft, ROUND((sum(ST_Area(the_geom::geography))/totalsm.total)::numeric,4) as percent, landuse, landuse_desc
+      FROM lots, totalsm
+      GROUP BY landuse, landuse_desc, totalsm.total
+      ORDER BY percent DESC
+    `;
+
+    return SQL;
+  }),
```

Comment:
I think at some point this should be handled in the route where all the data is prepared for the profile view itself before rendering. I think this works fine for now. There might need to be some "loading" logic in between route transitions.